devfile.yaml

```diff
@@ -36,14 +36,14 @@ components:
       endpoints:
         - name: http-8080
           targetPort: 8080
-  - name: outerloop-build
+  - name: image-build
     image:
       imageName: "dotnet-image:latest"
       dockerfile:
         uri: docker/Dockerfile
         buildContext: .
         rootRequired: false
-  - name: outerloop-deploy
+  - name: kubernetes-deploy
```

Comment:
no service here?

Reply:
dotnet has the service in a separate file. do we want to merge into a single one as well? to be consistent with others?